packages/webapp/pages/account/profile.tsx

```diff
@@ -54,6 +67,18 @@ const AccountProfilePage = (): ReactElement => {
   const [coverImage, setCoverImage] = useState<string>(user.cover);
   const currentCoverImage = coverImage || user.cover;
   const isMobile = useViewSize(ViewSize.MobileL);
+  const showExperienceLevel = useFeature(feature.experienceLevel);
+
+  // fetch user experience level
+  const { data } = useQuery<UserExperienceLevelData>(
+    generateQueryKey(RequestKey.UserExperienceLevel, user),
+    () =>
+      request(graphqlUrl, GET_USER_EXPERIENCE_LEVEL, {
+        id: user.id,
+      }),
+    { enabled: !!user && showExperienceLevel },
+  );
+  const experienceLevel = data?.user?.experienceLevel;
```

Comment:
Why do we do this as a seperate query, can't we just add this field to the profile query?

Reply:
We get all the other fields directly from the user object, as part of the `AuthContext`. Since this is the only place we use the field, it seems cleaner to me to add a specific query for this page, rather than query this field for AuthContext unnecessarily.

Reply:
That means adding it to boot, just FYI...

Reply:
We have `useProfile` which is used to get more complete version of user profile, but ti avoid refactoring of that page adding to boot for now is ok for me.

Reply:
Changed accordingly, we now get the experienceLevel field from boot.